### PR TITLE
docs: move adapter docs to site

### PR DESCRIPTION
**Pull request by benmccann**
This builds off the docs PRs from @dummdidumm: https://github.com/sveltejs/kit/pull/8501 and https://github.com/sveltejs/kit/pull/8500

I ignored all the types errors in the docs. I don't know if there's a better way to set them up

### CODE REVIEW

**Comment by dummdidumm** [comment]
I like this! I think I would rename the first doc to "fundamentals", but that's all. Let's wait on more opinions before doing this.

**Comment by Rich-Harris** [comment]
I think this is a good approach; will close #8500 and #8501 and do a proper edit of this branch tomorrow

**Comment by benmccann** [comment]
FYI:
>Bad link: adapters#supported-environments-node-js in 20-core-concepts/20-load.md
--
11:18:32.447 | Bad link: adapters#supported-environments-platform-specific-context in 25-build-and-deploy/10-building-your-app.md
11:18:32.460 | Bad link: adapters#supported-environments-platform-specific-context in types/ambient.d.ts

File: documentation/docs/25-build-and-deploy/40-adapter-node.md [hunk 1]
```diff
@@ -185,14 +185,10 @@ There's nothing built-in to SvelteKit for this, because such a cleanup hook depe
...
+	db.shutdown();
 }
 
 process.on('SIGINT', shutdownGracefully);
 process.on('SIGTERM', shutdownGracefully);
-```
-
-## Changelog
-
-[The Changelog for this package is available on GitHub](https://github.com/sveltejs/kit/blob/master/packages/adapter-node/CHANGELOG.md).
```
[the remark this answers is not in the document]
Comment (Rich-Harris) [reply documentation/docs/25-build-and-deploy/40-adapter-node.md]: Surely that link belongs in the README if anywhere? If you're looking for it it's very easy to find. We don't link to the license from the docs page so I'm not sure why we'd link to the changelog

File: documentation/docs/25-build-and-deploy/50-adapter-static.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
+---
+title: Static site generation
```
[the remark this answers is not in the document]
Comment (Rich-Harris) [reply documentation/docs/25-build-and-deploy/50-adapter-static.md:2]: I probably wouldn't associate 'static deployments' with SPAs either, so I think that'd be more loss than gain. And SPA doesn't necessarily imply static — you can have a client-rendered app that uses dynamic data. A lot of people use 'SPA' to mean 'SSR+SPA' these days anyway so it's become a fairly useless acronym.

I think we can signpost it with links in any case

File: .changeset/hungry-singers-tease.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+'@sveltejs/adapter-auto': patch
+'@sveltejs/adapter-cloudflare': patch
+'@sveltejs/adapter-node': patch
+'@sveltejs/adapter-static': patch
+'create-svelte': patch
+'@sveltejs/kit': patch
```
Comment (Rich-Harris) [inline .changeset/hungry-singers-tease.md:7]:
```suggestion
'@sveltejs/adapter-auto': patch
'@sveltejs/adapter-cloudflare': patch
'@sveltejs/adapter-cloudflare-workers': patch
'@sveltejs/adapter-netlify': patch
'@sveltejs/adapter-node': patch
'@sveltejs/adapter-static': patch
'@sveltejs/adapter-vercel': patch
```

File: .changeset/hungry-singers-tease.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+---
+
+docs: add more indepth docs on building your app
```
Comment (Rich-Harris) [inline .changeset/hungry-singers-tease.md:10]:
```suggestion
docs: move adapter docs to site
```

**Review by Rich-Harris — APPROVED** [review]

**Comment by dummdidumm** [comment]
Really like the structure! Wondering if "static site generation" is the right title since you also can build SPAs with it (a regular point of confusion). Maybe it's even benefitial to have a dedicated page for that? Could be something along the lines of "spas are x, you can create them like y, they are supported by various adapters. If the whole app is an spa, you can use the static adapter, see link for more info (links back to static site generation).

**Comment by Rich-Harris** [comment]
Re SPAs — https://github.com/sveltejs/kit/pull/8531#discussion_r1080553330 (you can build an 'SPA' with any adapter, since it describes rendering rather than static-ness). I could be persuaded that we need a new page for SPAs, distinct from build & deploy, as long as it includes 'why you probably shouldn't build an SPA', but i think we can tackle that separately

File: documentation/docs/25-build-and-deploy/50-adapter-static.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (benmccann) [reply documentation/docs/25-build-and-deploy/50-adapter-static.md:2]: Maybe there's some other term like "Statically served sites" that we could use?

**Comment by Rich-Harris** [comment]
I don't like how tall this section is relative to the others — it feels disproportionate. But I don't think there's an easy solution within the current structure so I guess we go along with it for now

image

**Pull request merged**